Data-Dictionary.md

```diff
@@ -0,0 +1,434 @@
+
+### Enrollment
+Rows in this table represent: A set of enrollment documents; append only
+
+#### Enrollment Columns
+- `enrollment_id`: Internal id of this enrollment document; integer
+- `document`: Original enrollment JSON document; json
+- `received`: When this enrollment document was received; timestamp
+- `processing_log`: Event log recording details of ETL into data warehouse; jsonb
+
+### FHIR
+Rows in this table represent: A set of [FHIR](fhir) documents; append only
+
+#### FHIR Columns
+- `fhir_ud`: Internal id of this fhir record set; integer
+- `document`: JSON document in [FHIR format](http://www.hl7.org/implement/standards/fhir/formats.html); json
+- `received`: When this FHIR document was received; timestamp
+- `processing_log`: Event log recording details of ETL into data warehouse; jsonb
+
+### Longitudinal
+Rows in this table represent: A set of longitudianl documents; append only
+
+#### Longitudinal Columns
+- `longitudinal_id`: Internal id of this longitudinal document; integer
+- `document`: JSON document created from a pre-processed Excel file; json
+- `received`: When this longitudinal document was received; timestamp
+- `processing_log`: Event log recording details of ETL into data warehouse; jsonb
+
+### Manifest
+Rows in this table represent: A set of manifest records; append only
+
+#### Manifest Columns
+- `manifest_id`: Internal id of this manifest document; integer
+- `document`: Manifest record as a JSON document; json
+- `received`: When this manifest document was received; timestamp
+- `processing_log`: Event log recording details of ETL into data warehouse; jsonb
+
+### Presence Absence
+Rows in this table represent: A set of presence/absence documents; append only
+
+#### Presence Absence Columns
+- `presence_absence_id`: Internal id of this presence/absence document; integer
+- `document`: Original presence/absence JSON document; json
+- `received`: When this presence/absence document was received; timestamp
+- `processing_log`: Event log recording details of ETL into data warehouse; jsonb
+
+### REDCap DET
+Rows in this table represent: A set of REDCap DET request data documents; append only
+
+#### REDCap DET Columns
+- `redcap_det_id`: Internal id of this REDCap DET document; integer
+- `document`: JSON document created from REDCap DET request data; json
+- `received`: When this REDCap DET document was received; timestamp
+- `processing_log`: Event log recording details of ETL into data warehouse; jsonb
+
+### Sequence Read Set
+Rows in this table represent: A set of references to sequence reads; append only
+
+#### Sequence Read Set Columns
+- `sequence_read_set_id`: Internal id of this sequence read set document; integer
+- `document`: Original sequence read set JSON document; json
+- `received`: When this sequence read set document was received; timestamp
+- `processing_log`: Event log recording details of ETL into data warehouse; jsonb
+
+## Shipping
```

Comment:
I know we had chatted on slack about which tables might make sense to include here. 
I wondered a bit about including views like fhir_encounter_details_v2 or observation_with_presence_absence_result_v3 or scan/uw enrollment/encounter views. 
I think adding them here might help with discoverability for db users. I'm not sure if those were already pretty self-explanatory or if there's logic in there that might be interesting to document, either in this or a followup PR. What are your thoughts on including those?

Reply:
Sure, taking another look at those it might make sense. I left `fhir_encounter_details_v2` off since it seemed like those columns are mostly just straight from REDCap so likely already documented up there. It might be good to transcribe those down here though so it is all in one place for discovery as you said. 

I believe I left `observation_with_presence_absence_result_v3` because it shares so many columns with `incidence_model_observation_v4` and I didn't feel like the extra columns it added (presence, device, assay_type, organism, target) needed additional documentation. 

For the SCAN/UW Enrollment/Encounter views I left them off with SCAN ending - although it occurs to me now that if we are into the analysis phase for some of that data it is probably a good idea to get shipping views documented for our external users. I'll take another look at those and see which ones might make sense to add.